.github/actions_info.md

```diff
@@ -47,3 +47,22 @@ Has the inputs:
   the matrix strategy.
 
 A matrix strategy will be adopted from the list of python and pytorch version.
+
+### Tests tutorials
+This workflow ([.github/workflows/tests_tutorials.yml](workflows/tests_tutorials.yml)) will setup
+an environment using the [env](#Env) action, clone the [tutorials repo](https://github.com/kornia/tutorials), and execute the tutorials.
+
+Use the actions:
+- `actions/checkout`
+- `.github/actions/env`
+
+Has the inputs:
+- `os`: (string, default: `ubuntu-latest`) the OS name same as supported by [gha](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#choosing-github-hosted-runners).
+- `python-version`: (json list of strings, default: `'["3.10"]'`) a string with
```

Comment:
Is there a great reason to default to Py310 which is 25% slower than Py311?

Reply:
Pytorch didn't fully support 3.11 yet